Fastlane/Fastfile

```diff
@@ -190,3 +190,16 @@ lane :pr_changes_contains_path do |options|
 
   changes_contains_path
 end
+
+lane :clean_up_release_from_tag do |options|
+  latest_tag = ENV['BITRISE_GIT_TAG']
+  sh "git push origin --delete release/#{latest_tag}"
+  sh "git push origin --delete #{latest_tag}"
```

Comment:
Not sure exactly, but do you know whether this will fail if the tag or release isn't found? If the first deletion fails, it should not prevent the second deletion from executing is what I'm worried about.

Reply:
It will fail, it won't prevent the the other one from running - or the other way round.

<img width="1282" alt="image" src="https://user-images.githubusercontent.com/5387088/170267951-ee085177-f2cc-43e2-ab9b-113003cf893e.png">

But I'll quickly confirm it in a ruby script, to check if the behavior is the same.

Reply:
Addressed in https://github.com/WeTransfer/WeTransfer-iOS-CI/pull/170/commits/2e4f6bc23d4a8b9b0d6794657e809e0b30cd95f3.

Reply:
Love it!